Phraends_Pkg/Frontend/Frontend.py

```diff
@@ -54,7 +53,7 @@ def news_tap_view(self):
         return 
     
     def news_tap_summary_section(self, ticker):

```

Comment:
I'm wondering should we put these functions in backend folder. What do you think?

Reply:
I forgot to add _view as a suffix for this function.

Why do you think it should stay in the backend since it's a sub-view of main_view?

Reply:
That's my fault, I thought the function was going to do some process. Sub-view is just fine. But there is still a  `get_yearlist` function, I think it should be put in backend folder.

Reply:
But for now, I think it's good to approve and merge

Reply:
I will chang the name, my dumb fault

Reply:
Don't forget the get_yearlist function! That is the only function should move to backend for now.